src/components/rating.css

```diff
@@ -0,0 +1,39 @@
+/* rating */
+.rating-wrapper {
+  padding: var(--sp-md);
+  border-radius: var(--radius-md);
+  box-shadow: var(--shadow-smooth);
+  width: 35rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.rating-header {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  align-items: center;
+  margin-bottom: 0.5rem;
+}
+
+.star-container {
+  display: flex;
+  gap: 1rem;
+  padding: var(--sp-xs) var(--sp-sm);
```

Comment:
very hard to understand this variable name. you can also make separate class for padding , like padding-sm , padding-lg etc.

Reply:
it is generic. sp for spacing